### PR TITLE
adding scotland script

### LINKED ISSUE #261: Produce Whole SDG Pop Estimates for Scotland
This involves producing the same calculation for England & Wales in the SDG script for Scotland. 

Steps for Developer:
- To produce an estimate for Scotland (without disaggregation's) 
- Include these data sources:
-- Scottish Usual Resident Population. 
-- Population Weighted Centroids.
--OA to LA look up

### PR DESCRIPTION
**Pull request by Antonio-John**
This is a new script which produces an estimate for Scotland. 

Steps to be able to run Scottish Data:
You'll need a few extra files, I'll detail which ones and where to save them here.
1) Scottish Usual Resident Population. This is the population per output area from the 2011 census. [KS101SC.csv](https://github.com/ONSdigital/SDG_11.2.1/files/9214008/KS101SC.csv) Needs to be placed in data/
2) Population weighted centroids are found [here.  ](https://www.spatialdata.gov.scot/geonetwork/srv/api/records/9d977a73-7884-4870-ae76-afccf8e6fae8)  On the link click on "Access the portal", there should be a download button on the top right. Download as a Zip file, extract the files and then place in data/pop_weighted_centroids/scotland/2011/
3) [OA to LA lookup](https://geoportal.statistics.gov.uk/datasets/ons::postcode-to-output-area-to-lower-layer-super-output-area-to-middle-layer-super-output-area-to-local-authority-district-november-2021-lookup-in-the-uk/about) needs to be placed in data/oa_la_mapping/scotland/2021

closes #261

### CODE REVIEW

**Review by paigeh1 — CHANGES_REQUESTED** [review]
Happy with the code. Really minor change which I have added in the review, then will be happy to approve it.

File: SDG_scotland.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    total_df_dict[local_auth] = la_results_df_out
+
+# every single LA
+all_la = pd.concat(total_df_dict.values())
+
+all_la.to_csv("Scotland_results.csv", index=False)
+
+end = time.time()
+
+print(f"This took {(end-start/60)} minutes to run")
```
[the remark this answers is not in the document]
Comment (Antonio-John) [reply SDG_scotland.py:178]: good spot!

[the remark this answers is not in the document]
Comment (Antonio-John) [reply SDG_scotland.py:178]: have changed to `print(f"This` took {(end-start)/60} minutes to run")``

**Review by paigeh1 — APPROVED** [review]
Happy to approve this now, great work!

**Pull request merged**